azure/packages/test/scenario-runner/src/utils.ts

```diff
@@ -56,6 +56,17 @@ export function createAzureTokenProvider(
     });
 }
 
+export function createInsecureTokenProvider(
+    tenantKey: string,
+    userID?: string,
+    userName?: string,
+): InsecureTokenProvider {
+    return new InsecureTokenProvider(tenantKey, {
```

Comment:
The issue here may be with literal and know properties. Can you try defining user outside of the call, instead of passing literal?